assets/less/zenodo-rdm/globals/site.overrides

```diff
@@ -8,6 +8,10 @@
 @navbarBackgroundImage: linear-gradient(to right,#0047A8 0,#2BBCFF 100%);
 @searchButtonColor: #f2711c;
 
+.ui.header.text-muted {
```

Comment:
shouldn't this come from invenio-app-rdm?

Reply:
https://github.com/inveniosoftware/invenio-app-rdm/pull/1941